index.html

```diff
@@ -0,0 +1,104 @@
+<html>
+<head>
+	
+	<link rel="stylesheet" href="https://use.typekit.net/rkk2hbc.css">
+	
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Roboto+Flex:opsz@8..144&display=swap" rel="stylesheet">
+	
+	<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
+	
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	
+	<title>Chloe's Portfolio</title>
+	
+	<link rel="stylesheet" type="text/css" href="style.css">
+	
+</head>
+
+<body>
+	
+<nav>
+	
+	<div class="navbar">
+		
+		<a href="#about-me.html" class="contact-link">About</a>
+		<a href="#portfolio.html" class="contact-link">Portfolio</a>
+		<a href="#contact.html" class="contact-link">Contact</a>
+		
+	</div>
+	
+</nav>
+
+<main>
+	
+	<div class="home-body">
+		
+		<h1 class="chloe-smart">Chloe Smart</h1>
+		<p class="software-developer">SoftwareDeveloper</p>
+		<img class="spots" src="spots-blue.png" alt="abstract spots">
+		
+	</div>
+	
+	<div class="about-me-section" id="about-me.html">
+		
+		<div class="about-text">
+			
+			<h2 class="about-me-title">About Me</h2>
+		
+			<p class="about-me">Hello! I'm Chloe, I am an aspiring Software Developer. Previously I have worked as a Printed Textile Designer, and for the past few years I have owned my own illustration & design business.</p>
+			<p class="about-me">I first got into coding when building & designing my own website for my business, I loved thinking about how to create the best experience for the user & problem solving how to make my website work to a high quality</p>
+				
+		</div>
+		
+		<img src="me.jpg" alt="A photo of myself, doing some illustration in my favourite cafe" class="me">
+		
+	</div>
+	
+	<div class="project-section" id="portfolio.html">
```

Comment:
the id should be `portfolio` not `portfolio.html`, i find it curious you named it that way haha

Reply:
same for the other ones ending in .html